### PR TITLE
Address all 3 issues :)

### PR DESCRIPTION
**Pull request by MainShayne233**

### CODE REVIEW

**Comment by ardavis** [comment]
Why'd you close it?

File: app/views/tasks/forms/_form.html.haml [hunk 1]
```diff
@@ -18,6 +18,9 @@
             = link_to 'Create New Tag', new_tag_modal_path(task: @task), class: 'dropdown-item', remote: true
           %li
             .divider
+          %li
+            = link_to '#', value: nil, class: 'tag_option dropdown-item' do
+              %span &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (ardavis) [inline app/views/tasks/forms/_form.html.haml:23]: Josh Johnson needs to see this line. He will love it.

Comment (MainShayne233) [inline app/views/tasks/forms/_form.html.haml:23]: Hahaha I was going to break it out into a partial to cover my shame.

**Comment by MainShayne233** [comment]
I closed it because I forgot to address some of the other fixes that I planned on.

**Pull request closed without merging**